#! /bin/sh -e

# DP: Search new '/usr/lib/$(DEB_BUILD_GNU_TYPE)'-style directories for
# DP: libraries.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -298,6 +298,8 @@
         lib_dirs = self.compiler.library_dirs + [
             '/lib64', '/usr/lib64',
             '/lib', '/usr/lib',
+            '/lib/%s' % os.getenv('DEB_BUILD_MULTIARCH'),
+            '/usr/lib/%s' % os.getenv('DEB_BUILD_MULTIARCH')
             ]
         inc_dirs = self.compiler.include_dirs + ['/usr/include']
         exts = []
